tycho-its/src/test/java/org/eclipse/tycho/test/baseline/BaselineMojoTest.java

```diff
@@ -139,4 +224,78 @@ private Verifier getBaselineProject(String project) throws Exception {
 		verifier.addCliOption(project + "/pom.xml");
 		return verifier;
 	}
+
+	private static void modifyProjectPom(Path projectRoot, Consumer<Model> pomModifier) throws IOException {
+		File pomFile = projectRoot.resolve("pom.xml").toFile();
+		Model model = new DefaultModelReader().read(pomFile, null);
+		pomModifier.accept(model);
+		new DefaultModelWriter().write(pomFile, null, model);
+	}
+
+	private static void modifyProjectManifest(Path projectRoot, Map<String, String> newEntries) throws IOException {
```

Comment:
Why not just a `Consumer<Manifest>`? It seems much more flexible to do anything one wants with the manifest. Also the name "newEntries" is confusing when it actually can replace / delete items.

Reply:
Changed it to use a `Consumer<Manifest>`.